src/test/kotlin/no/nav/syfo/slowtests/repository/UtsattOppgaveRepositoryImpTest.kt

```diff
@@ -84,6 +84,28 @@ open class UtsattOppgaveRepositoryImpTest : SystemTestBase(){
        Assertions.assertNotEquals(firstId, secondId)
     }
 
+    @Test
+    fun `oppdaterer oppgaver med enhet`() {
+        val enhet = "Test123"
+        val utsattOppgave = UtsattOppgaveEntitet(
+            inntektsmeldingId = "id3",
+            arkivreferanse = "arkivref",
+            fnr = "fnr",
+            aktørId = "aktørId",
+            sakId = "sakId",
+            journalpostId = "journalpostId",
+            timeout = now().minusHours(1),
+            tilstand = Tilstand.Utsatt
+        )
+
+        repository.opprett(utsattOppgave)
+
+        utsattOppgave.enhet = enhet
+
+        val oppdatertEnhet = repository.oppdater(utsattOppgave).enhet
+        Assertions.assertEquals(enhet, oppdatertEnhet)
```

Comment:
Siden repo.oppdater bare returnerer det samme objektet som blir putta inn så får du ikke testa at eneheten leses ut igjen fra databasen her.

Du må bruke findAll() for å asserte at koden klarer å lese ut igjen enehten fra databasen.

Reply:
Oppdatert nå